mercury/apps/notebooks/management/commands/watch.py

```diff
@@ -35,6 +35,7 @@ def handle(self, *args, **options):
             notebook_id = task_init_notebook(
                 options["notebook_path"], is_watch_mode=True
             )
+            self.stdout.write(self.style.SUCCESS(f"http://127.0.0.1:8000/app/{notebook_id}"))
```

Comment:
When adding a helpful printout on terminal please always think about users that are going to read that message. Do you think that your message is clear enough that beginner user will understand?